openpype/tools/publisher/widgets/help_widget.py

```diff
@@ -12,6 +12,7 @@ class HelpButton(QtWidgets.QPushButton):
     def __init__(self, parent):
         super(HelpButton, self).__init__(parent)
         self.setObjectName("CreateDialogHelpButton")
+        self.setFocusPolicy(QtCore.Qt.NoFocus)
```

Comment:
The object on it's own cannot modify focus. The window must be first shown to be able to define focus.

Reply:
The logic works fine however - what do you mean? :)

This is the focus policy - not the active focus.